client/package.json

```diff
@@ -188,8 +188,8 @@
     "test": "echo 'Run \"test-all\" to execute all client-side tests, or pick a specific test target: \"test-jest\", \"test-wdio\", \"test-wdio-ie\", \"test-e2e\".' ; exit 1",
     "test-all": "yarn run test-jest && yarn run test-wdio && yarn run test-wdio-ie && yarn run test-e2e",
     "test-e2e": "export SERVER_URL=${SERVER_URL:-'http://localhost:8180'} ; NODE_ENV=test node $(npm bin)/ava --config ava.config.js",
-    "test-wdio": "export SERVER_URL=${SERVER_URL:-'http://localhost:8180'} ; NODE_ENV=test wdio --baseUrl ${SERVER_URL} ./config/wdio.config.js",
     "test-jest": "export ANET_DICTIONARY=../anet-dictionary.yml ; NODE_ENV=test jest --testPathPattern='tests/(actions|reducers|models|utils)'",
+    "test-wdio": "export SERVER_URL=${SERVER_URL:-'http://localhost:8180'} ; export ANET_DICTIONARY=../anet-dictionary.yml; NODE_ENV=test wdio --baseUrl ${SERVER_URL} ./config/wdio.config.js",
```

Comment:
wdio tests should get the dictionary through a request to the server, not through a local file; revert this change.